lib/marked.js

```diff
@@ -460,7 +460,8 @@ var inline = {
   code: /^(`+)\s*([\s\S]*?[^`])\s*\1(?!`)/,
   br: /^ {2,}\n(?!\s*$)/,
   del: noop,
-  text: /^[\s\S]+?(?=[\\<!\[_*`]| {2,}\n|$)/
+  text: /^[\s\S]+?(?=[\\<!\[_*`]| {2,}\n|$)/,
+  moretext: /^(?:[^\s([{]_[A-Za-z0-9]|[A-Za-z0-9]_\S)/
```

Comment:
This regex looks suspiciously specific. Why not `\S` instead of `[A-Za-z0-9]`, for instance?